src/nft/components/explore/Carousel.tsx

```diff
@@ -37,36 +32,32 @@ const CarouselCardContainer = styled.div`
 
 const CarouselItemCard = styled(a.div)`
   ${carouselHeightStyle}
-  ${carouselItemStyle}
 
   display: flex;
   justify-content: center;
-  padding: 3px 32px 32px 32px;
-
+  padding: 4px 12px 32px;
   position: absolute;
   will-change: transform;
+
+  @media screen and (min-width: ${({ theme }) => theme.breakpoint.lg}px) {
+    padding: 4px 32px 32px;
+  }
 `
```

Comment:
Idea:

Bring back `carouselItemStyle`, set `padding-left` and `padding-right` to `12px` and override them in a media query (not the whole `padding`, just left/right).

Reply:
carouselItemStyle was just the padding rules, and one of the components that used it overwrote them anyway. I don't think the re-use benefit is meaningful in this case

Reply:
i do think abstracting this might make it harder to follow